### DISCUSSION TITLE
Looking for NFT generator web app Hello im looking for developer who can help me to build an own NFT Generator app with smart contract builder integrated.  Unanswered Question

### DISCUSSION THREAD

**Discussion by alimttth** [community/community#38486] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi! First, I'd like to say that your emulator is so cool! I've been using it since May of 2022 and I can't even tell you how impressive this emu is! The quality of the games in 4k plus the Vulkan system you operate in your emulator is out of this world! Thank you for all this really. I stopped using Duckstation for some time though, and I ended up deleting it altogether, then around September, I decided I wanted to reinstall it for the second time so off I went, however, this time some titles that I love, one of them being Silent Hill 1, began to crash while playing. It automatically took me out of the game with no apparent reason, something that had never happened back then in May, (first time I tried Duckstation on my MacBook Air M1) plus some other issues like in some video clips suddenly everything freezes out. I honestly would like to continue to use this emulator as in my opinion it's the best PS1 emulator under the sun and I love the design and the aesthetics of the games thanks to Duckstation and thanks to all of you who have made this possible. I hope you can work this/these issue(s) out. I don't know why this happens but I am 100 percent sure it did not happen before. Especially with this one game Silent Hill among some other titles too. Thank you for your time! Please let me know how I can contribute to this project so that I can help you somehow. Kindest regards! Aaron.

**Comment by david-engelmann** [comment, marked as answer]
I'm not sure this is the best location for this inquiry, this discussion section is mainly for github questions and suggestions. It's probably best to move this to the specific repo you are asking about!

**Comment by omararabeh28** [comment]
كيف احصل على رابط للدخول الى NFTs
